%% All authors must submit their articles at
%% \href{http://www.pnascentral.org/cgi-bin/main.plex}{PNAScentral}. If
%% you are using Overleaf to write your article, you can use the
%% ``Submit to PNAS'' option in the top bar of the editor window.

\documentclass[11pt,twocolumn,twoside,lineno]{pnas-new}
% Use the lineno option to display guide line numbers if required.

\usepackage[utf8]{inputenc}

\templatetype{pnasresearcharticle} % Choose template 
% {pnasresearcharticle} = Template for a two-column research article
% {pnasmathematics} %= Template for a one-column mathematics article
% {pnasinvited} %= Template for a PNAS invited submission

\title{Genotypic variation in a foundation tree alters ecological
  network structure of an associated community}

% Use letters for affiliations, numbers to show equal authorship (if
% applicable) and to indicate the corresponding author

\author[a,b,1]{Matthew K. Lau}
\author[b]{Louis J. Lamit}
\author[c]{Rikke R. Naesbourg}
\author[d]{Stuart R. Borrett}
\author[e]{Matthew A. Bowker}
\author[a]{Thomas G. Whitham}

%% Include department, institution, and complete address, with the
%% ZIP/postal code, for each author. Use lower case letters to match
%% authors with institutions, as shown in the example. 
%% Authors with an ORCID ID may supply this information at submission.

\affil[a]{Department of Biological Sciences and Merriam-Powell Center
  for Environmental Research, Northern Arizona University, Flagstaff,
  AZ 86011, USA}
\affil[b]{Harvard Forest, Harvard University, 324 N Main St,
  Petersham, MA 01366, USA}
\affil[d]{Department of Biology, Syracuse University, 107 College
  Place Syracuse, NY 13244, USA}
\affil[e]{University of California Berkeley, Berkeley, CA, USA}
\affil[f]{Department of Biology and Marine Biology, University of
  North Carolina Wilmington, 601 South College Road, Wilmington, NC,
  28403, USA}
\affil[g]{School of Forestry, Northern Arizona University, Flagstaff,
  AZ 86011, USA}

% Please give the surname of the lead author for the running footer
\leadauthor{Lau} 
% Use letters for affiliations, numbers to show equal authorship (if
% applicable) and to indicate the corresponding author
% Please add here a significance statement to explain the relevance of your work

%% Authors must submit a 120-word maximum statement about the
%% significance of their research paper written at a level
%% understandable to an undergraduate educated scientist outside their
%% field of speciality. The primary goal of the Significance Statement
%% is to explain the relevance of the work in broad context to a broad
%% readership. The Significance Statement appears in the paper itself
%% and is required for all research papers.



\significancestatement{Evolution occurs in the context of ecosystems
  comprised of complex ecological networks. Research at the interface
  of ecology and evolution has primarily focused on pairwise
  interactions among species and have rarely included a genetic
  component to analyses. Here, we use a long-term common garden
  experiment to reveal the effect that genotypic variation can have on
  networks of lichens that occur on the bark of a foundation tree
  species. We found that lichen interaction network structure is
  genetically based and primarily driven a tree trait, bark
  roughness. These findings demonstrate the importance of genetic
  variation and evolutionary dynamics in shaping ecological networks
  as evolved traits. In particular, this study points to the
  importance of assessing the effect of foundation species genetics on
  the structure of interactions, given that interaction network
  structure has systems-level properties that could affect the
  response of these communities to selection.}

% Please include corresponding author, author contribution and author
% declaration information

\authorcontributions{M.L. and L.L. conceived the study, M.L. and
  L.L. conducted the field work, R.N.  assisted in lichen
  identifications, M.L. wrote the first draft of the manuscript,
  S.B. and T.W. contributed substantively to the conceptual
  development, T.W. established the common garden. All authors
  contributed to revisions of the manuscript.}
\authordeclaration{The authors have no conflicts of interest.}
\correspondingauthor{\textsuperscript{1}Dr. Matthew K. Lau. E-mail:
  matthewklau@fas.harvard.edu}

% Keywords are not mandatory, but authors are strongly encouraged to
% provide them. If provided, please include two to five keywords,
% separated by the pipe symbol, e.g:
\keywords{networks $|$ heritability $|$ community $|$ genetics $|$
  lichen $|$ cottonwood $|$ Populus $|$ common garden}

\begin{abstract}
%% Please provide an abstract of no more than 250 words in a single
%% paragraph. Abstracts should explain to the general reader the major
%% contributions of the article. References in the abstract must be
%% cited in full within the abstract itself and cited in the text.

%% Currently: 234 words 12November2019

Biological evolution occurs in the context of complex networks of
interacting species in which natural selection defines the structure
of ecological networks. Fundamental to this evolutionary process is
the discovery of a genetic basis to ecological network
structure. Although previous work has demonstrated that tree genotype
contributes to interaction network structure at the scale of forest
stands, the contribution of tree genetics to localized interaction
networks at the scale of individual trees has not yet been
explored. To test the degree to which tree genetics can contribute to
network structure we conducted quantitative modeling of interaction
networks. We constructed networks of epiphytic lichens associated with
individual trees that were a part of a long-term experimental common
garden of genotypes of (\textit{Populus angustifolia}), a foundation
species. We found two primary results. First, tree genotype
significantly predicted lichen network similarity, i.e. clonal
replicates of the same genotype tended to support more similar lichen
networks, using multiple network metrics. Second, although multiple
species were observed repeatedly in the interaction networks, the
effect of tree genotype was primarily focused on variation of the
interactions of one lichen species, \textit{Caloplaca
  holocarpa}. Third, one of the examined tree traits, bark roughness,
was both predicted by tree genotype and correlated with lichen network
similarity, supporting a mechanistic pathway from a variration in a
heritable tree trait and the genetically based variation in lichen
network structure. We conclude that tree genotype can influence not
only the relative abundances of organisms but also the interaction
network structure of associated organisms. Given that variation in
network structure can have consequences for the dynamics of
communities through altering the stability of the system and
modulating or amplifying perturbations, these results have important
implications for the evolutionary dynamics of ecosystems.


\end{abstract}

\dates{This manuscript was compiled on \today}
\doi{\url{www.pnas.org/cgi/doi/10.1073/pnas.XXXXXXXXXX}}

\begin{document}

%% Many authors find it useful to organize their manuscripts with the
%% following order of sections; Title, Author Affiliation, Keywords,
%% Abstract, Significance Statement, Results, Discussion, Materials
%% and methods, Acknowledgments, and References. Other orders and
%% headings are permitted.

%% PNAS generally uses a two-column format averaging 67 characters,
%% including spaces, per line. The maximum length of a Direct
%% Submission research article is six pages and a Direct Submission
%% Plus research article is ten pages including all text, spaces, and
%% the number of characters displaced by figures, tables, and
%% equations.  When submitting tables, figures, and/or equations in
%% addition to text, keep the text for your manuscript under 39,000
%% characters (including spaces) for Direct Submissions and 72,000
%% characters (including spaces) for Direct Submission Plus.

%% Authors may use 1- or 2-column equations in their article,
%% according to their preference.

%% To allow an equation to span both columns, use the
%% \verb|\begin{figure*}...\end{figure*}| environment mentioned above
%% for figures.

%% Note that the use of the \verb|widetext| environment for equations
%% is not recommended, and should not be used.

%% \begin{figure*}[bt!]
%% \begin{align*}
%% (x+y)^3&=(x+y)(x+y)^2\\
%%        &=(x+y)(x^2+2xy+y^2) \numberthis \label{eqn:example} \\
%%        &=x^3+3x^2y+3xy^3+x^3. 
%% \end{align*}
%% \end{figure*}


%% \begin{table}%[tbhp]
%% \centering
%% \caption{Comparison of the fitted potential energy surfaces and ab
%% initio benchmark electronic energy calculations}
%% \begin{tabular}{lrrr}
%% Species & CBS & CV & G3 \\
%% \midrule
%% 1. Acetaldehyde & 0.0 & 0.0 & 0.0 \\
%% 2. Vinyl alcohol & 9.1 & 9.6 & 13.5 \\
%% 3. Hydroxyethylidene & 50.8 & 51.2 & 54.0\\
%% \bottomrule
%% \end{tabular}
%% \addtabletext{nomenclature for the TSs refers to the numbered
%% species in the table.}
%% \end{table}

%% References should be cited in numerical order as they appear in
%% text; this will be done automatically via bibtex,
%% e.g. \cite{belkin2002using} and
%% \cite{berard1994embedding,coifman2005geometric}. All references
%% should be included in the main manuscript file.

%% PNAS must be able to archive the data essential to a published
%% article. Where such archiving is not possible, deposition of data
%% in public databases, such as GenBank, ArrayExpress, Protein Data
%% Bank, Unidata, and others outlined in the Information for Authors,
%% is acceptable.


%% Only TIFF, EPS, and high-resolution PDF for Mac or PC are allowed
%% for figures that will appear in the main text, and images must be
%% final size. Authors may submit U3D or PRC files for 3D images;
%% these must be accompanied by 2D representations in TIFF, EPS, or
%% high-resolution PDF format.  Color images must be in RGB (red,
%% green, blue) mode. Include the font files for any text.

%% Figures and Tables should be labeled and referenced in the
%% standard way using the \verb|\label{}| and \verb|\ref{}| commands.

%% Figure \ref{fig:frog} shows an example of how to insert a
%% column-wide figure. To insert a figure wider than one column,
%% please use the \verb|\begin{figure*}...\end{figure*}|
%% environment. Figures wider than one column should be sized to 11.4
%% cm or 17.8 cm wide. Use \verb|\begin{SCfigure*}...\end{SCfigure*}|
%% for a wide figure with side captions.

%% Authors should submit SI as a single separate PDF file, combining
%% all text, figures, tables, movie legends, and SI references.  PNAS
%% will publish SI uncomposed, as the authors have provided it.
%% Additional details can be found here:
%% \href{http://www.pnas.org/page/authors/journal-policies}{policy on
%% SI}.  For SI formatting instructions click
%% \href{https://www.pnascentral.org/cgi-bin/main.plex?form_type=display_auth_si_instructions}{here}.
%% The PNAS Overleaf SI template can be found
%% \href{https://www.overleaf.com/latex/templates/pnas-template-for-supplementary-information/wqfsfqwdiscujtsd}{here}.
%% Refer to the SI Appendix in the manuscript at an appropriate point
%% in the text. Number supporting figures and tables starting with S1,
%% S2, etc.

%% Authors who place detailed materials and methods in an SI Appendix
%% must provide sufficient detail in the main text methods to enable a
%% reader to follow the logic of the procedures and results and also
%% must reference the SI methods. If a paper is fundamentally a study
%% of a new method or technique, then the methods must be described
%% completely in the main text.


\maketitle \thispagestyle{firststyle}
\ifthenelse{\boolean{shortarticle}}{\ifthenelse{\boolean{singlecolumn}}{\abscontentformatted}{\abscontent}}{}

% If your first paragraph (i.e. with the \dropcap) contains a list
% environment (quote, quotation, theorem, definition, enumerate,
% itemize...), the line after the list may have some extra
% indentation. If this is the case, add \parshape=0 to the end of the
% list environment.

%% \dropcap{T}his PNAS journal template is provided to help you write
%% your work in the correct journal format.  Instructions for use are
%% provided below.

%% Note: please start your introduction without including the word
%% ``Introduction'' as a section heading (except for math articles in
%% the Physical Sciences section); this heading is implied in the
%% first paragraphs.

%% Introduction

%% - Evolution occurs in complex ecological networks, which have been
%% treated primarily from a pairwise perspective
%% - Network ecology suggests that network architecture has important
%% implications for ecological and evolutionary dynamics
%% - Community genetics has shown that intraspecific variation impacts
%% communities and ecosystems
%% - What we know about networks in a community genetics context is
%% Bailey 2009 (tri-trophic chain), Bridgeland ????, Smith 2012
%% (geographic mosaic), Lamit 2015 (JoE), Lau 2016 (bipartite), Keith
%% 2017 (bipartite).
%% - None of these studies have looked at networks of organisms at a
%% scale in which population genetics of trees might impact entire
%% networks of interacting organisms, hence this study
%% - Hypothesis: based on the genetic similarity rule, genetically
%% similar individuals should tend to have similar communities and
%% based on past research with network ecology, they should also have
%% similar interaction networks


\dropcap{E}volution occurs in the context of complex ecological
networks. Initially, evolution in a community context was focused on
examples of highly co-evolved pairs of species (e.g. Darwin's famous
prediction of the Sphinx Moth and Christmas Orchid)
\cite{Wade2007TheCommunities}. However, studies of diffuse
co-evolution (\textit{sensu} \cite{Janzen1980SpecificityForest})
\cite{StinchcombeCanHerbivores, Strauss2008EvolutionSize}, geographic
mosaics of co-evolution \cite{Thompson2013} and community genetics
\cite{Whitham2006a} have provided an in-road for ecological network
approaches \cite{Smith2015b, Keith2017, Lau2016GenotypicEvolution} to
illuminate a more complex perspective of the interface between
ecological and evolutionary dynamics. There is now evidence to support
that selection tends to occur among groups of species
\cite{Wade2007TheCommunities} favoring the development of small webs
\cite{Guimaraes2007InteractionNetworks, Gomez2009LocalMosaic} and that
genetic variation and phylogenetic relatedness contributes to
variation in community assembly \cite{Crutsinger2016} and species
interactions \cite{Whitham2006a, Bailey2009a, Moya-Larano2011}, which
shapes the ecological interaction networks \cite{Rezende2007}.

\textit{Define foundation species.}

Community genetics studies \cite{Lamit2015a} have shown that genetic
variation in foundation species \cite{Ellison2005} plays a significant
role in defining distinct communities of interacting organisms:  such
as, endophytes, pathogens, lichens, arthropods, and soil
microbes. Multiple studies have now demonstrated that genetic
variation influences numerous functional traits (e.g., phytochemical,
phenological, morphological) produces a multivariate phenotype
\cite{holeski2012} that contributes to variation in associated
communities \cite{Bailey2009a}. The importance of genetic variation in
structuring ecological systems was recently reviewed
\cite{DesRoches2018TheVariation}, and not only were many instances of
strong genetic effects found in many ecosystems but the effect of
intraspecific variation was at times greater than
\textit{inter}-specific variation.

Additional work has provided support for the hypothesis that not only
does composition vary among genetically distinct genotypes of
foundation species but that it also impacts the structure of species
interactions. However, studies in the network ecology literature
generally do not include a genetic component \cite{Lau2017a}; and,
community genetics studies have primarily focused on community
composition in terms of the abundance of species
\cite{DesRoches2018TheVariation}. There are studies that have examined
the effects of genetic variation on trophic chains, such as several in
plant-associated communities (including \textit{Populus},
\textit{Solidago}, \textit{Oenothera}, \textit{Salix})
\cite{Bailey2005ImportanceInteractions, Johnson2008, Smith2011,
  Smith2015b, Barbour2016GeneticComplexity} and generally found that
increasing genotypic diversity leads to increased trophic
complexity. Only two other studies, that we are aware of, have
explicitely examined the effect of genotypic variation on the
structure of interaction networks between tree individuals and
assoicated herbivores \cite{Lau2015a, Keith2017} and both found that
genotypic diversity generates increased network modularity
(i.e. compartmentalization). However, both of these studies were at
the scale of forest stands and, therefore, were not able to observe
replicated networks in order to statistically test for genetic effects
on network structure.


\parshape=0

Here, we investigate how genetic variation in a foundation tree
species determines the structure of a network of interactions among a
community of tree associated lichen species. Previous studies have
examined aspects of networks \citep{Barbour2019TraitCommunities}. Here
we examine the genetic basis of network structure on a community of
sessile lignicolous (i.e. bark) lichens on cottonwood trees. Using a
long-term (20+ years), common garden experiment with replicated
individuals of known genetic identity and a naturally established
stand of \textit{Populus angustifolia}. We focused on a model
community of 9 epiphytic lichen species, as previous research has
demonstrated significant compositional responses of epiphytes to
genotypic variation \citep{Winfree2011, Zytynska2011}. In addition,
the life-history characteristics of lichens, having highly localized,
direct contact interactions and slow population turnover rates,
facilitated the assessment of interactions among lichen species on
individual trees. We hypothesize that in natural systems evolution
occurs in a community context involving interactions of complex
networks of interacting species \cite{Lau2015a, Keith2017,
  Thompson2013, Bascompte2006}.  If correct, we expect to find that
network structure is genetically based in which different plant
genotypes support different interaction networks and that these
interactions networks can function as indicators of ecological
dynamics important for conserving biodiversity.  Applying a
probability-theory based network modeling approach, we constructed a
set of interaction network models for the lichens associated with
individual trees. Using these models, we then examined the genetic
basis of the structure of these ecological networks. Based on previous
community genetics studies, particularly \citep{Bangert2006} which
proposed the community similarity rule, we hypothesize that trees will
vary in some phenotypic traits and those trees of the same genotype
will tend to have similar traits leading to similarities in lichen
network structure.


\matmethods{

\subsection*{Study System}

The study was conducted along the Weber River, UT (USA), which is a
cottonwood (\textit{Populus} spp.) dominated riparian
ecosystem. Although two native species, \textit{Populus angustifolia}
(James) and \textit{Populus fremontii} (S. Watson), occur here and are
known to hybridize, only pure or advanced generation backcrosses of
\textit{P. angustifolia} were sampled. Bark lichens have been
extensively studied in this system and provide an ideal system in
which to observe and model lichen interaction networks, as their
sessile nature permits accurate identification of individuals
\cite{Lamit2011}.

A long-term, common garden experiment was used to isolate the effect
of tree genotype from the effect of the localized microenvironment
associated with each individual and spatial
autocorrelation. Established in 1992, asexually propagated clones of
genotyped \textit{P. angustifolia} individuals were obtained from wild
collections and planted in fully randomized design at the Ogden Nature
Center, Ogden, UT. From the population of established individuals in
the common garden, we chose a total of ten genotypes, replicated
between 3 and 8 times each, for sampling.



\subsection*{Bark Lichen and Trait Observations}


On each tree, presence or absence of each lichen species was assessed
in 50 total 1 cm$^2$ cells arrayed in a checkerboard pattern. Given
the small size and sessile nature of lichens, we were able to rapidly
assess lichen interactions by quantifying thalli in close
contact. Sampling was restricted to the northern aspect of the trunk
to maximize the abundance of lichen and control for the effect of
trunk aspect. Two adjacent 100 cm$^2$ quadrats centered at 50 cm and
95 cm from ground level were sampled (Fig~\ref{fig:lichen_sampling} A
and B). The observed lichen community included (abbreviations are
given for species present in study): Xg = \textit{Xanthomendoza
  galericulata}, Xm = \textit{X. montana}, Ch = \textit{Caloplaca
  holocarpa}, Cs = \textit{Candelariella subdeflexa}, Rg =
\textit{Rinodina glauca}, Lh = \textit{Lecanora hagenii}, Pm =
\textit{Phyciella melanchra}, Pa = \textit{Physcia adscendens}, Pu =
\textit{Physcia undulata}. Several other species were not obesrved in
the present study but are known to occur in this region:
\textit{Phaeophyscia orbicularis}, \textit{Phaeophyscia ciliata},
\textit{Melanelia subolivacea}, \textit{Meanelia elegantula}.


The cell size and checkerboard sampling pattern was chosen to isolate
the individuals in each cell. In a previous survey of lichen thallus
size in this common garden, we had observed a median thallus size of
0.12 $\pm$ 0.001 cm$^2$ (1 S.E.) (see Supporting Information). Based
on the median thallus size, we expected thalli observed in each cell
to generally be spatially independent of thalli present in other cells
but exposed to similar micro-environmental conditions created by the
bark and the location of the sampling area on an individual
tree. Therefore, we were confident in treating the cell-wise
observations in quadrats as independent with respect to lichen-lichen
interactions.



We also measured several bark traits for each tree: including, bark
roughness, condensed tannin, carbon and nitrogen concentrations and
pH. \textbf{ADD METHODS FROM JAMIE}.



\begin{figure*}[ht]
\centering
\includegraphics[width=\linewidth]{lcn_sampling.pdf}
\caption{The communities of bark lichens were observed in a common
  garden of replicated genotypes of narrowleaf cottonwood trees
  (\textit{P. angustifolia}) at the Ogden Nature Center (Ogden,
  UT). Lichens were sampled within a fixed area (100 cm$^2$) on
  individual trees at two heights, 500cm and 95cm from the ground (A
  and B, respectively). (C) a photo of a typical community of bark
  lichen species interacting on the trunk of a cottonwood tree,
  including one of the more abundant species, \textit{Xanthomendoza
    galericulata}, in the center. (D-K) shows the other lichen species
  observed, respectively:  \textit{X. montana}, \textit{Candelariella
    subdeflexa}, \textit{Rinodina} sp., \textit{Caloplaca holocarpa},
  \textit{Physcia adscendens}, \textit{Phyciella melanchra},
  \textit{Physcia undulata} and \textit{Lecanora hagenii}. Photo
  Credits: L.J. Lamit (A-C) and R.R. Naesbourg (D-K).}
\label{fig:lichen_sampling}
\end{figure*}


\subsection*{Lichen Network Modeling and Analysis}

For each tree, repeated observations of lichen were made in order to
construct replicated interaction networks for each genotype. We
conducted a modified sampling procedure originally developed by
\citep{Lamit2015a} with the addition that we quantified the presence of
lichen in the 1 cm$^2$ cells on individual trees of
\textit{P. angustifolia}. Unipartite networks were generated using the
conditional probabilities of each species pair, i.e. the probability
of observing one species given an observation of another species
$P(S_i | S_j)$, based on the method developed by
\citep{Araujo2011}. To calculate conditional probabilities, we
quantified the individual probabilities of species occurrences
$P(S_i)$ and the joint probability of co-occurrences $P(S_i,S_j)$
using the frequencies of each species and their co-occurrences. We
were then able to calculate the conditional probabilities of each
species pair as $P(S_i|S_j) = \frac{P(S_i,S_j)}{P(S_j)}$, based on the
axioms of probability. This yielded a matrix that could possibly be
asymetric, i.e. $P(S_i|S_j)$ does not have to be equal to
$P(S_j|S_i)$. Another important property of this matrix is that the
diagonal ($S_{ii}$) was equal to one for all species present and zero
for species that were not observed in any cell.

We then applied an analytical procedure to remove non-significant
links between species. This procedure determines if the joint
probability of a species pair (i.e. $P(S_i,S_j)$) is different from
zero (Fig.~\ref{fig:conet_method}).  Here, a confidence interval
$CI_{95\%}$ is calculated as as $CI_{95\%} = E[S_iS_j] * Z_{95\%} *
\sqrt{V(S_iS_j)}$, where the expected frequency of co-occurrences
E($S_iS_j$) is the total number of cells surveyed ($N$) times the
independent probabilities of each species $P(S_i) * P(S_j)$,
$Z_{95\%}$ is the Z-score for 95\% from a Z-distribution and the
expected variance of $E(S_iS_j)$ is the total number of cells times
the expected probability of $S_iS_j$ and its compliment
(i.e. $V(S_iS_j) = N * E[P(S_i,S_j)] * (1 - E[P(S_i,S_j)])$). If the
observed number of co-occurrence falls outside of the confidence
interval, the joint probability $P(S_i,S_j)$ is determined to be equal
to the product of the individual probabilities (i.e. $P(S_i) \dot
P(S_j)$), and the conditional probability reduces to the individual
probability of that species $P(S_i)$. Therefore, unless the
co-occurrence of a species pair falls outside the confidence interval,
the probability that the observation of one species given the other is
no different than simply observing that species alone. This enables us
to remove links from a given network by re-scaling the resulting
conditional probabilities by subtracting the individual probabilities
from the conditional probabilities (i.e. how different the conditional
probability is from the independent probability), which makes any
species with a non-significant conditional probability zero. The
resulting matrix ($\mathbf{D} = D_{ij}$) can be interpreted as how one
species impacts another with zero being no effect and values less than
or greater than zero interpreted as negative and positive effects,
respectively. Here, we will refer to this matrix ($\mathbf{D}$) as an
interaction matrix with the properties that it can be asymetric
(i.e. $P_{ij}$ does not necessarily equal $P_{ji}$), and the diagonal
($P_{ii}$) is zero (i.e. a species does not influence it's own
probability of being observed). 

\begin{figure*}[ht]
\centering
\includegraphics[width=\linewidth]{lcn_araujo_method.pdf}
\caption{Lichen interaction networks were constructed by conducting
  field observations in 1 cm$^2$ cells within a 100 cm$^2$ grid on each
  tree using a checkerboard pattern (grey cells). Thus, a set of $N$
  total cell observations were recorded for each tree with the
  presence or absence of each species recorded for each cell. Applying
  the probability-based network modeling method adapted from
  \cite{Araujo2011}, we calculated the conditional probabilities,
  $P(S_i|S_j)$, for all species pairs and removed (i.e. set equal to
  zero) species pairs whose joint probabilities, $P(S_i S_j)$, were
  not significant using a confidence interval based comparison of
  their observed co-occurrence frequency, $S_iS_j$, to that expected
  due to chance alone, $E[P(S_iS_j)] = P(S_i) P(S_j)$, and
  $P(S_i|S_j)$ reduces to $P(S_i)$, the observed individual
  probability of species $S_i$. In the context of these networks,
  asymetry and positive/negative valued connections are distinct
  quantities. In-coming and out-going connections can be interpreted
  as ``influenced by'' and ``influenced'', respectively; while
  positive and negative should be seen as one species increasing or
  decreasing, respectively, the probability of another species'
  occurrence.}
\label{fig:conet_method}
\end{figure*}


\subsection*{Network Metrics}

To quantify the structural variation of lichen networks we calculated
several metrics at both the node and whole-network level. For
individual nodes (i.e. species) in each network, we calculated both
the degree \eqref{eq:1} and the centrality. We also calculated two
similar global network metrics: degree and centralization. The first
was network degree, which is a count of the total number of links in a
network. As the networks contained not only positive and negative
connections, as well as directional connections (both in-coming and
out-going), we calculated the same network metrics for all
combinations of these types of connections in each network. Although
there are many more possible network metrics that could have been
examined, we chose to focus on a restricted set for the sake of
clarity. Also, degree and centrality form the basis of many other
network metrics. 

\textbf{ADD EQUATIONS FOR METRICS}

\begin{equation}
  \sum{x_i}\label{eq:1}
\end{equation}

\begin{itemize}
\item Node degree
\item Node centrality
\item Network degree
\item Centralization 
\item In vs out 
\item Pos vs neg 
\end{itemize}

To calculate separate metrics for positive and negative links, we
applied methods for calculating the centrality accounting for the
sign differences \cite{Everett2014NetworksTies}. We used the \texttt{signnet}
package version ????, which is available at ???.


\subsection*{Statistical Analyses, Software and Data}

We used a combination of parametric and non-parametric, permutation
based frequentist statistical analyses to test for the effects of
genetic variation on lichen communities and their interaction
networks. To assess the effect of genotype on univariate responses, we
used additive, random effects models with Restricted Maximum
Likelihood (REML). We used a combination of Least Squares Regression,
Analysis of Variance (ANOVA) and correlation tests to quantify and
test for the relationship among other variables. Bark roughness,
lichen cover and species richness were square-root transformed to meet
the assumptions of homogeneity of variance and normality for these
tests.

For multivariate response variables, such as lichen community
composition and network structure, we used distance based multivariate
statistical approaches, including Permutational Analysis of Variance
(PERMANOVA) and Mantel tests. To quantify the similarity of lichen
networks among individual trees, we calculated the pairwise Euclidean
distance of the $\mathbf{D}$ interaction matrices among all pairs of
trees.

For visualization of multivariate patterns, we used Non-metric
Multi-Dimensional Scaling (NMDS) \cite{ecodist} to produce
dimensionally reduced ordinations of these multi-variate responses and
fitted vectors for continuous predictor variables to the ordinated
values \cite{vegan}. Using random initial configurations with a
maximum of 500 iterations and a change in stress threshold of less
than 10$^{-12}$. Final configurations has the lowest stress with at
most a stress level of 0.10.

For each network, we also calculated metrics that measure different
structural aspects. Although there are many other metrics, for the
sake of simplicity we focus on a subset that represent several
interesting features of network structure (see \citep{Lau2017a}). We
calculated the number of interactions or ``links'' in each network,
which provides a measure of the size of the network \citep{Lau2015a,
  Borrett2014EnaR:Analysis}. We also calculated the centralization of
each network, which measures the evenness of the distribution of
interactions among the species in the network \cite{sna}. In a network
with a low level of centralization species have similar amount of
interaction in the network, while a network with a high level of
centralization tends to have one or small number of species that
interact with other species. We used a related function to calculate
the centrality of each species (i.e. node level centrality) in each
network as well.

For all tests where genotype was used as a predictor, we quantified
the heritability of the response variable. Because the trees in the
garden were clonal replicates of each genotype, we calculated
broad-sense heritability, which is the genotypic variance divided by
the total phenotypic variance \cite{Conner2004ATextbook}. This can be
interpreted as a measure of the phenotypic variance due to genotypic
variation. We also apply this to the community genetics context as the
variance in \textit{extended} phenotypic variance due to genotypic
variation \cite{DawkinsTheGene}. For the multivariate analyses, where
we employ PERMANOVA, we followed the methods of
\citep{Shuster2006COMMUNITYSTRUCTURE} to adjust the degrees of freedom
for unbalanced genotype replicates.

All code and data for the project are openly available online. Code
and data are available at \url{github.com/ecgen/comgen}. The project
is also archived via Zenodo at \url{zenodo.com/doiXXXXXX}. All
analyses were conducted using the programming language R version 3.6.1
(R Development Core Team 2019).

\begin{figure}[ht]
\centering
\includegraphics[width=\linewidth]{cn_onc.pdf}
\caption{Lichen networks varied in structure among tree
  genotypes. Network diagrams of the mean lichen interaction matrices
  averaged for all trees and for several individual genotypes (996,
  WC5 and 1008) showing a range of interaction network
  structure. Directionality (arrowheads) and sign (red = negative,
  black = positive) of interactions are shown as edges between species
  (abbreviated by the first letter of the genus and specific epithet),
  which are scaled by their magnitude. The sign of the interaction is
  indicative of greater (positive) or lesser (negative) paired
  occurrences than expected relative to the overall frequency of
  occurrence of each species. Ecologically, the links in the network
  are likely the product of multiple types of interactions
  (e.g. mutualism, parasatism, competition, facilitation) that could
  vary over both space and time.}
\label{fig:geno_nets}
\end{figure}
}

\showmatmethods{} % Display the Materials and Methods section

\section*{Results}

%% New results Fri May  1 19:00:26 EDT 2020

Tree genotype influenced lichen network structure and multiple lichen
network metrics were heritable.  Tree genotype significantly predicted
the structural similarity of lichen networks (PERMANOVA:
Pseudo-$F_{9,27}$ = 3.58, $H^2$ = 0.41, \textit{p-value} = 0.0537)
(Fig.~\ref{fig:h2_plot}).  Overall network level metrics responded
signficantly to tree genotype (Table~\ref{tab:h2_net}), including
network degree (\textit{RLRT} = 3.52, $H^2$ = 0.32, \textit{p-value} =
0.0255) and centralization including both in-coming and out-going
links (\textit{RLRT} = 4.04, $H^2$ = 0.33, \textit{p-value} = 0.0184)
or when separated into in-coming only (\textit{RLRT} = 3.9852, $H^2$ =
0.3309, \textit{p-value} = 0.0190) or out-going only (\textit{RLRT} =
3.8615, $H^2$ = 0.3193, \textit{p-value} = 0.0205).  Metrics including
only positive links also showed a signficant effect of tree genotype,
including positive degree (\textit{RLRT} = 3.6925, $H^2$ = 0.3242,
\textit{p-value} = 0.0229), positive in-going centralization
(\textit{RLRT} = 4.4812, $H^2$ = 0.3487, \textit{p-value} = 0.0142)
Metrics calculated with negative links were not significant, including
degree (negative) (\textit{RLRT} = 0.0327, $H^2$ = 0.0318,
\textit{p-value} = 0.3859) and both in-coming (negative)
(\textit{RLRT} = 0.3304, $H^2$ = 0.1057, \textit{p-value} = 0.2508)
and out-going centralization (negative) (\textit{RLRT} = 0.0862, $H^2$
= 0.0513, \textit{p-value} = 0.3446).

\begin{figure*}[ht]
\centering
\includegraphics[width=\linewidth]{h2_plot.pdf}
\caption{The similarity of lichen networks varied among tree
  genotypes. A. The plot shows genotype centroids of NMDS ordinated
  (R$^2$ = 0.999, stress = 0.008) lichen networks ($\pm$ 1
  S.E.). Genotype centroids that are closer together tend to have more
  similar lichen network structure.  B. Plot showing the standardized
  ($\frac{x - \bar{x}}{\sigma}$) means ($\pm$ 1 S.E.)  for the two of
  the genetically based lichen network metrics: overall degree
  (i.e. total number of links) and centralization, which is a measure
  of the dominance of one species in the network.}
\label{fig:h2_plot}
\end{figure*}

\input{h2_reml_net}

The genetic response of network centralization was driven by variation
in \textit{Caloplaca holocarpa}. Centrality varied significantly among
species ($F_{8, 324}$ = 7.99, $R^2$ = 0.16, \textit{p-value} $<$
0.0001). \textit{Caloplaca holocarpa} centrality was the main species
to exhibit a significant response to tree genotype in terms of
positive centrality for both the in-coming (\textit{RLRT} = 3.61,
$H^2$ = 0.32, \textit{p-value} = 0.0240) and out-going (\textit{RLRT}
= 3.13, $H^2$ = 0.30, \textit{p-value} = 0.0327) perspectives, but not
for either negative centrality metrics in-coming (\textit{RLRT} = 0,
$H^2$ = 0, \textit{p-value} = 1) or out-going (\textit{RLRT} = 0,
$H^2$ = 0, \textit{p-value} = 0.4543). None of the other species'
centralities showed a genotypic response (Supplementary
Table~\ref{tab:sppcen}) with the exception of \textit{X. montana}
(\textit{RLRT} = 2.92, $H^2$ = 0.32, \textit{p-value} = 0.0375);
however, the centrality of \textit{X. montana} was much lower overall
relative to \textit{C. holocarpa} and the variation in
\textit{X. montana} centrality was restricted to two genotypes
(Fig.~\ref{fig:geno_sppcen}).



\begin{figure*}[ht]
\centering
\includegraphics[width=\linewidth]{geno_sppcen.pdf}
\caption{Dot-plots showing the mean (dot) and $\pm$ 1 SE of in-degree
  (A) and out-degree (B) centrality for two species,
  \textit{C. holocarpa} and \textit{X. montana}. \textit{Caloplaca
    holocarpa} centrality was highly variable among
  genotypes. \textit{Xanthomendoza montana} centrality, both in- and
  out-degree, was only non-zero for two genotypes, and only out-degree
  centrality displayed a significant response to genotype.}
\label{fig:geno_sppcen}
\end{figure*}

\textit{Add transformations of variables to methods.}

Genotype indirectly influenced lichen network centralization via the
genetically based variation in bark roughness. The percent cover of
rough bark (\textit{RLRT} = 4.8526, $H^2$ = 0.3221, \textit{p-value} =
0.0113) and condensed tannins (\textit{RLRT} = 3.0522, $H^2$ = 0.3205,
\textit{p-value} = 0.0343) both displayed significant responses to
tree genotype. None of the other bark traits, pH (\textit{RLRT} =
0.00, $H^2$ = 0.00, \textit{p-value} = 1.0000) or carbon-nitrogen
Ratio (\textit{RLRT} = 0.0000, $H^2$ = 0.0000, \textit{p-value} =
1.0000), showed a signficant response to tree genotype and none other
than bark roughness were correlated with network similarity
(Table~\ref{tab:cn_trait_perm}); therefore, we focused
our analysis on bark roughness. We found that bark roughness was
signficantly correlated with network similarity (PERMANOVA:
Pseudo-$F_{1,32}$ = 13.029, $R^2$ = 0.26, \textit{p-value} = 0.0096)
and other lichen network metrics, including negative correlations with
overall network degree ($df$ = 35, $t$ = -2.13, $r$ = -0.34,
\textit{p-value} = 0.04) and centralization ($df$ = 35, $t$ = -2.52,
$r$ = -0.39, \textit{p-value} = 0.02). In other words, trees with more
similar levels of bark roughness tended to have lichen interaction
networks with similar structure. To quantify the genetic bases of this
effect of bark roughness on network structure, we used the residual
values from regressions of network degree and centralization in tests
of the effect of tree genotype and found no significant effect of tree
genotype for either degree (\textit{RLRT} = 0.00, $H^2$ = 0.00,
\textit{p-value} = 1.0000) or centralization (\textit{RLRT} = 0.00,
$H^2$ = 0.00, \textit{p-value} = 1.0000), suggesting that the observed
relationship between bark roughness and lichen network structure was
largely genetically based (Fig.~\ref{fig:br_net}).

\input{cn_trait_perm.tex}

\begin{figure*}[ht]
\centering
\includegraphics[width=\linewidth]{br_net.pdf}
\caption{Bivariate plots of the relationship between bark roughness
  and three network metrics: A) degree and B) centralization. Each
  plot displays the genotype mean $\pm$ 1 S.E. for both variables and
  a least-squares regression calculated using the genotype means.}
\label{fig:br_net}
\end{figure*}


%% In addition to including your tables within this manuscript file, PNAS
%% requires that each table be uploaded to the submission separately as a
%% “Table” file.  Please ensure that each table .tex file contains a
%% preamble, the \verb|\begin{document}| command, and the
%% \verb|\end{document}| command. This is necessary so that the
%% submission system can convert each file to PDF.


\section*{Discussion}


We found that tree genotype influenced lichen network structure in the
experimental cottonwood forest. Network similarity and metrics of
network structure tended to be more similar on trees of the same
genotype. Generally, this genetic effect was manifested in positive
interactions and largely driven by \textit{C. holocarpa}. The
genetically based trait, bark roughness, was the only trait observed
to effect network variation, largely via shifts in positive in-coming
and out-going interactions. Bark roughness has been demonstrated
previously to be under strong genetic control \cite{Bdeir2017}, and
bark roughness has also previously been shown to be an important tree
trait influencing bark lichens \cite{Lamit2015a}; however this is the
first demonstration of a link from genes to lichen network structure.
As such these results have important implications for the potential
influence of genetically based variation in ecosystems with networks
of interacting species.

There are important functional ramifications of genetically based
variation in network structure. First, even if the composition of the
communities is the same among individuals and genotypes, interactions
may not be. We didn't observe compositional differences using the same
data from which the lichen networks were derived. If we only had our
composition dataset from this study, we would have concluded no
response of the lichen community to tree genotype, even though the
underlying interactions among lichen species does vary among
genotypes. Community composition of lichen has previously been
observed to be different among tree genotype in the same experimental
garden, though this was observed with a larger sampling of total area
and quadrats per tree. Regardless, this could result in a situtation
in which abundance based investigations of community-level genetic
effects may miss important variation in the interactions among
individuals in these communities, leading to an underestimate of
genetic effects in ecosystems. It is possible that these underlying
differences in interactions among lichen could lead to differences in
community composition at a future point in time, however, this is not
needed for evolutionary dynamics to play out.

\textit{Integrate these comments on sampling from LJL}

\begin{itemize}
\item It's important to note that total cover, richness and
  composition were not responsive to tree genotype in the current
  study, in contrast with previous studies.
\item The previous study used samples from both the northern and
  southern aspects of each tree; whereas, the current study only
  observed lichen on the northern aspect. 
\item Also, our current results are likely different from the previous
  study because the current study selected genotypes that tended to
  have bark lichen, with the interest of focusing on generating
  networks for comparison.

\end{itemize}


The second will take a little more finesse to work out but I think it
is easily possible. I am concerned that composition, total cover and
richness are not responsive to genotype. I don’t think this is an
issue for your paper, since you bring up the really cool idea that
networks could be different but composition does not have to be. I
think there are good legitimate reasons associated with sampling
style, for example using frequency in 1cm2 vs continuous \% cover
values for composition and \% cover calculations, which could lead to
different results. 

\begin{itemize}
\item It is possible the frequency method over estimating cover
  values.
\item Also, like you bring up, just focusing on the north side and not
  including the south side would change things since the south side
  had far less lichen cover. 
\item The main aim was to collect from genotypes that had a decent
  amount of lichen cover in order to ensure you could build networks.
\end{itemize}

I think these are both reasons that your overall \% cover values are
much higher than what we got in our 2015 ecology paper. For your data,
I calculated the range in the mean \% total lichen cover among
genotypes as ~60-93\% cover, whereas the range that we report in the
2015 paper is 0.86 – 18.73\%. Again, these lower values I think are
due to some of the sampling differences, which is totally fine, we
should just probably articulate this more clearly (you have a part in
the discussion where I added some thoughts about this). 

Perhaps towards the end you picked up a couple other genotypes but I
think the main set was genotypes with a fair amount of lichen. 

In our 2015 paper a main idea and finding was that genotypes varied in
how good they were as substrates for lichen community formation, with
some genotypes having very little cover and richness while others had
much more. This was then related to a shift in composition that was
like the accumulation of species into the formation of communities as
they assemble….basically, some genotypes allowed for the fast assembly
of communities and some did not. 

The attached NMDS from our paper shows the basic pattern supporting
this explanation communities on the left had very little lichen and
very little community overall while those on the far right had much
more of a fully assembled community. What I think is that the pool of
genotypes we sample for networks might be primarily from genotypes
that are on the right side of the ordination, which is why there is
not heritable variation in cover, richness or composition among
them. This idea is also supported by the fact that your genotypes
ranged from 3.88-5.33 richness on average, whereas the pool in the
2015 paper was 1.33-6.00 despite the fact that the 2015 sampling
covered much greater area on a tree.

I don’t think this is a big issue; since you wanted to characterize
communities, it makes sense that we chose a random pool of genotypes
from the upper distribution of cover and richness so that we could
have enough of a community to build networks. However, it seems that
we should make this clear when describing sampling methods and also
help it inform that paragraph in the discussion when you mention the
results from the 2015 paper. 

Let me know what you think. I can help with the wording in parts, if
you would like. I am also thinking it would be a useful exercise for
me to select out the composition, richness and \% cover data from the
exact same quadrants as you sampled and do an analysis on them to see
if they also show not difference; this would allow us to know if it is
sampling approach or the total pool of genotypes that is driving the
differences.



Second, following on the previous point, genetic diversity could be
influencing the stability of communities through the effects on the
structure of interactions. Some network structures are likely to be
more stable, either in response to disturbance or via self-organized
dynamics. For example, centralized networks, although more effecient,
are theorized to be more susceptible to targeted attacks on the center
of the network. For example, consider a forest with two genotypes that
support lichen communities that are similar in total abundances of
each species but differ in terms of the structure. Extensions of game
theory to evolutionary biology have demonstrated that network
structure can lead to variation in evolutionary dynamics. Some
structures tend toward dominance and dampening of selection, while
others lead to amplification of selection (Newman). One class of
networks that are theorized to have amplifying effects on networks
have "star" shapes with one or a few species at the center and
radiating interactions out from the central core (Leiberman). This is
structurally what we have observed with the networks that tend to
occur on some of the genotypes in our study, i.e. the more centralized
networks. It is possible that these more centralized networks could
function as hot-spots of evolutionary dynamics resulting from the
amplifying effect the network structure fostered on that tree
genotype.

Altering the structure of interaction networks presents a means for
genetic effects to be magnified within the system of interacting
species. For example, \citep{Keith2017} showed that the genetics based
interactions of aphid resistant and aphid susceptible trees resulted
in different interaction networks of their associated arthropod
communities composed of 139 species. At the scale of ecosystems,
trophic networks or food webs direct and control the rates of energy
and nutrient flux \cite{Borgatti2006}. Furthermore, in a
predator-prey-plant study, Smith \cite{Smith2011}, showed that the
interactions among species across trophic levels depended on plant
genotype. Also, work by \citep{Toju2017, Toju2016, Toju2014a} observed
consistent patterns of centralized interactions of species modules
focused around hubs of plant-fungal interactions. In other words, a
small number of plant and fungal symbionts tended to have
disproportionate numbers of interactions with other species and likely
are the drivers in determining community assembly, structure and
dynamics. 

\textit{Review papers that use network analysis to identify foundation
species.}

\textit{Make a bigger deal of the application of the genetic
  similarity rule and more similar genotypes supporting more similar
  interaction networks.}

There are several important points to consider with regard to the
generalization of the observed genetically based response of the
lichen networks. Body size and sessile nature of lichen important to
observing genotype responses. As bark lichen individuals do not move,
but grow in a largely two dimensional plane, these communities and
their interactions occur in the highly localized context of the tree's
bark surface. Lichen individuals are many orders of magnitude smaller
than the tree individual and the life-span of a tree is many times
that of a lichen. For these reasons, any genetic effects on these
communities is not dampened by the movement of individuals and the
mixing of the effect of different tree genotypes on the lichen
community, as might occur for more mobile species (e.g. insects and
birds). We only looked at lichen, other species whose distribution,
abundance or interactions respond to tree genotype, such as epiphytic
plants (e.g. moss and liverworts), algae or insects, could be playing
a role. Other traits could also be playing a role, such as traits that
are correlated with bark roughness, such as micro-aspect, albedo,
moisture, etc.



%% Interestingly, structure of interactions within a functional group
%% have typically been abstracted or ignored in ecological network
%% studies. A good example of this is the bipartite network literature,
%% which explicitely does not include interactions within groups in the
%% analysis of ecological networks, such as plant-pollinator
%% interactions. The basis for this is that interactions within groups do
%% not contribute substantively to the dynamics of the system. Here, we
%% clearly show a mechanistic basis for the structure of an associated
%% community of lichen. 


\textit{Add more text here on positive and negative interactions of
  lichen from the literature.} 


\textit{Elaborate on how many species might be supported by each
  lichen thallus including the symbionts and other species.}


\textit{Elaborate on the importance of identifying the species/node
  level patterns in network structure response to genotype.}


\textit{What is the relationship between tree growth, bark roughness
  and disruption of the lichen community? Tree grow and the bark
  expands over time, causing furrows. }

\textit{Does bark roughness increase habitat and decrease interactions?}



Bark roughness could possibly be serving the role that other lichen
play in facilitating the success of new propagule attachment and the
growth of establishing thalli. This is supported by the patterns
overall being positive, including \textit{C. holocarpa} centrality. We
did not observe specific microscopic dynamics, such as photobionts,
mycobionts, endolichenic fungi and bacteria, but variation in these
underlying interactions could also be playing a role. Although we can
not rule out the possibility that other unmeasured tree traits or
organisms correlated with bark roughness are underlying the observed
patterns in bark lichen network structure, substantial research
supports the importance of genetically based tree traits for
communities and ecosystems \cite{DesRoches2018TheVariation}, and in
particular bark roughness for bark lichen communities \cite{Bdeir2017,
  Lamit2011, Lamit2015a},





Although our study was conducted with a community of lichens, these
results should be generalized to other groups of diverse organisms
around the world that also exhibit significant genetic signals at the
community level \cite{Rowntree2011, Whitham2012}. In the face of the
high degree of complexity and potential context dependency of
ecological processes, the current study points to the utility of
considering the spatial and temporal scales of interactions, as
discussed to some in previous studies \cite{Bangert2006, Zook2010,
  Zytynska2012}. In the present study, we found that community
assembly processes, such as environmental filtering and species
interactions, are genetically based. This is likely due, in part, to
the large difference in the differences in size and longevity of the
lichen and cottonwood individuals with the trees determining the
environment in which the lichen occur. We suggest that future work
would be aided by determining these modules within the biotic
community that include species with similar differences in body-size
and time-scales. As heritable variation is the raw material for
natural selection to act upon, a genetic basis for interaction network
structure indicates evolutionary dynamics should be considered at the
community level and that conserving genetic variation is important to
consider in efforts to restore or preserve complex species
interactions and their associated ecosystem functions
\cite{Evans2013}.  With such findings, it appears that we are closer
to understanding the evolutionary drivers of Darwin's entangled bank
and the interconnectedness of species in complex communities.


Future work should consider the potential influence on evolutionary
dynamics of the associated communities. The network of interactions of
species that are strongly influenced by a foundation species, could
amplify the effects of genotype, this serves as a means for genetic
effects to increase rather than diffuse through an ecosystem either
through space or over time, as has been proposed in the construction
of the genetic diffusion hypothesis. Altered abundances can lead to
differences in interactions Genotype effects on abundances of
individual abundances may cancel out. Specifically for asexually
reproducing species, such as many lichen are, shifting interaction
frequencies could lead to evolutionary outcomes, given the potential
to take-up symbionts and genetic material from thalli that they come
into contact with. Altering interaction frequencies could result in
differences in rates of the exchange of genetic materials among lichen
that could then be passed on to vegetative and possibly sexually
produced reproductive propagules. The larger scale (stand or region)
effects of these "evolutionary units" on each tree would depend on the
connectivity and rate of movement of propagules among trees per the
geographic mosaic of co-evolution hypothesis \cite{Barbour2009c,
  Thompson2013}.


\textit{Add evolutionary unit to the larger narrative. That is mainly
  that identifying how evolution might be acting on multiple species
  is useful for conservation and managment.}


\textit{Discuss the impacts of P. betae removal on network structure
  per Lau 2016 and Keith 2017. Also, what about Barbour 2016.}


\subsection*{Other studies that should be discussed:}

\begin{itemize}
\item Multiple plant traits shape the genetic basis of herbivore
  community assembly. Synthesis: Taken together, our results support
  that the genetic basis of herbivore community assembly occurs
  through a suite of plant traits for different herbivore species and
  feeding guilds (Barbour 2015). Also discuss Lamit 2015 and Holeski's
  multivarite phenotype paper.
\item Linking plant genes to insect communities:  Identifying the
  genetic bases of plant traits and community composition. Synhesis:
  These findings support the concept that particular plant traits are
  the mechanistic link between plant genes and the composition of
  associated insect communities (Barker 2019).
\end{itemize}



%% \textbf{TGW:  Is there any adaptive component to the tree in having
%%   certain lichen communities?  e.g., can they feed back to affect tree
%%   performance in some way or is this a passive outcome of a trait that
%%   affects bark for other adaptive reasons and lichens are passive
%%   players that tag along for the ride?  I could envision that lichens
%%   covering the bark of a tree act as a barrier between insects and
%%   pathogens, much like ectomycorrhizae cover fine roots as a first
%%   line of defense by invading microorganisms.  Uptake of N that gets
%%   passed to the tree??}


%% \textbf{LJL: I agree that there is a general overarching theme that
%%   evolution occurs in a community network context, but I’m not sure
%%   that we should state that as our main hypothesis. It seems more that
%%   this is a fundamental foundation for our work. The hypothesis is
%%   more what we are testing directly, but we don’t test this
%%   directly. I guess I don’t want to give the impresison that our
%%   communities are necessarily the result of each species evolving into
%%   its place in the community on these tree genotypes (although I do
%%   understand this as Shuster et al 2006’s fundamental explanation for
%%   why we see different communities on different genotypes; I don’t
%%   necessarily agree that this is the only reason we woudl see
%%   different communities on dif genotypes). Most of these are pretty
%%   generalist lichens, which could be found on other decidous trees in
%%   the surounding city or natural areas. I would look at it more like
%%   an assembling of lichen species into unique configurations on
%%   genetically different substrates. There may be some selection for
%%   different genotype of lichen during the community assembly process
%%   but we can’t really tell that just by differences in species
%%   abundances or coocurneces. I guess to me the evolutionary context
%%   that is more direclty related to this work is that the tree genotype
%%   is a central controller (indeed a sort of hub species in the
%%   network) of network structure. By anchoring the lichen network to
%%   tree genotype (and variation among networks to variation among tree
%%   genotypes) , our study highlights the possibility that natural
%%   selection acting on the trees may have an extended consequence for
%%   the network structure of organisms living on the trees…the extra
%%   thing we add to the field is that we show interaction networks are
%%   sensitive to genotype. I doubt the lichens have a direct effect on
%%   tree fitness, but favorability of some tree genotypes over others
%%   during natural selection will then go on to favor and disfavor
%%   certain lichen communities of different network structures. By being
%%   sensitive to tree genotype, the lichen community networks are
%%   passive riders on the waves of evolutionary dynamics that occur
%%   within the tree species they inhabit.}

%% \textbf{TGW: might be good to \cite papers on competition in lichens or
%%   other organizing factors to back up the least expected statement.
%%   as epiphytes we might not expect them to care.}

%% \textbf{TGW: I think we need to emphasize the long-term nature of our
%%   common garden study as very few common garden studies of lichens
%%   likely exist. Any refs on this? If true might want to mention this
%%   up front in intro.}

%% \textbf{MKL: Environmental filtering is evidenced by species richness,
%%   but also possibly species interaction varying based on environment
%%   as networks varied in terms of sign and magnitude as well.}

%% \textbf{MKL: The effect of bark roughness on network similarity was
%%   primarily genetically based, and there are likely other factors at
%%   play.}

%% \textbf{Discussion of network implications for stability with genetics.}


%% Please include your acknowledgments here, set in a single
%% paragraph. Please do not include any acknowledgments in the
%% Supporting Information, or anywhere else in the manuscript.



\acknow{This work was supported by the National Science Foundation grant
(DEB-0425908) and Integrative Graduate Research Traineeship (IGERT)
fellowships for M.L. and L.L. The Ogden Nature Center staff helped to
maintain the common gardens. Lichen sampling was supported by Todd
Wojtowicz, Luke Evans and David Solance Smith.}

\showacknow{} % Display the acknowledgments section

\bibliography{lichen_network_genetics}

\newpage
\section*{Supplementary Materials}

\setcounter{figure}{0}
\setcounter{table}{0}

\subsection*{Tables}

\newpage
\input{cn_perm.tex}

\newpage
\input{h2_reml.tex}

\newpage
\input{cn_trait_perm.tex}

\newpage
\input{sppcen.tex}

\newpage
\input{cormat.tex}

\newpage
\input{com_perm.tex}

\newpage
\subsection*{Figures}

\begin{figure*}[ht]
\centering
\includegraphics[width=\linewidth]{xg_size.pdf}
\caption{}
\label{fig:xg_size_app}
\end{figure*}

\begin{figure*}[ht]
\centering
\includegraphics[width=\linewidth]{spac_geno.pdf}
\caption{Species area curve by genotype.}
\label{fig:spac_geno_app}
\end{figure*}


\begin{figure*}[ht]
\centering
\includegraphics[width=\linewidth]{spp_cen.pdf}
\caption{}
\label{fig:spp_cen_app}
\end{figure*}

%% \begin{figure*}[ht]
%% \centering
%% \includegraphics[width=\linewidth]{spp_cen_in.pdf}
%% \caption{}
%% \label{fig:spp_cen_in_app}
%% \end{figure*}

%% \begin{figure*}[ht]
%% \centering
%% \includegraphics[width=\linewidth]{spp_cen_out.pdf}
%% \caption{}
%% \label{fig:spp_cen_out_app}
%% \end{figure*}

\end{document}

%% \subsection*{Supporting Information (SI)}

%% Authors should submit SI as a single separate PDF file, combining
%% all text, figures, tables, movie legends, and SI references.  PNAS
%% will publish SI uncomposed, as the authors have provided it.
%% Additional details can be found here:
%% \href{http://www.pnas.org/page/authors/journal-policies}{policy on
%% SI}.  For SI formatting instructions click
%% \href{https://www.pnascentral.org/cgi-bin/main.plex?form_type=display_auth_si_instructions}{here}.
%% The PNAS Overleaf SI template can be found
%% \href{https://www.overleaf.com/latex/templates/pnas-template-for-supplementary-information/wqfsfqwyjtsd}{here}.
%% Refer to the SI Appendix in the manuscript at an appropriate point
%% in the text. Number supporting figures and tables starting with S1,
%% S2, etc.

%% Authors who place detailed materials and methods in an SI Appendix
%% must provide sufficient detail in the main text methods to enable a
%% reader to follow the logic of the procedures and results and also
%% must reference the SI methods. If a paper is fundamentally a study
%% of a new method or technique, then the methods must be described
%% completely in the main text.

%% \subsubsection*{SI Datasets} 

%% Supply Excel (.xls), RTF, or PDF files. This file type will be
%% published in raw format and will not be edited or composed.

%% \subsubsection*{SI Movies}

%% Supply Audio Video Interleave (avi), Quicktime (mov), Windows Media
%% (wmv), animated GIF (gif), or MPEG files and submit a brief legend
%% for each movie in a Word or RTF file. All movies should be
%% submitted at the desired reproduction size and length. Movies
%% should be no more than 10 MB in size.


%% \subsubsection*{3D Figures}

%% Supply a composable U3D or PRC file so that it may be edited and
%% composed. Authors may submit a PDF file but please note it will be
%% published in raw format and will not be edited or composed.

